data/auxiliary/gather/ldap_query/ldap_queries_default.yaml

```diff
@@ -81,15 +81,15 @@ queries:
       - groupType
       - memberof
   - action: ENUM_ORGUNITS
-    description: 'Dump info about all known organizational roles in the LDAP environment.'
+    description: 'Dump info about all known organizational units in the LDAP environment.'
```

Comment:
Just confirming, isn't this correct? 👀

Reply:
The changes were that organization roles here were being used as the description vs organization units. The ENUM_ORGUNITS enumerates organizational units which are different types of objects than organization roles.

This fixes this by aligning the description appropriately as I seem to have made a copy paste mistake when I was rearranging the order of a few of these.